README.md

```diff
@@ -258,6 +258,12 @@ This object defines templates (functions) that are used for displaying parts of
 
 `suggestion` is a function that receives one argument, a suggestion to be displayed. It is used when rendering suggestions, and should return a string, which can contain HTML. :warning: **Caution:** because this function allows you to output arbitrary HTML, you should [make sure it's trusted](https://en.wikipedia.org/wiki/Cross-site_scripting), and accessible.
 
+#### `dropdownArrow` (default: A rectangle pointing down)
+
+Type: `Function`
+
+A function that gets passed an object with the property `className` (`{ className: '' }`) and should return either a string (ATTENTION this will NOT be escaped. Be sure to only pass in safe values) or a (P)React element.
```

Comment:
👍 

Can you change this to be 

```
A function that gets passed an object with the property `className` (`{ className: '' }`) and should return a string of HTML or a (P)React element. :warning: **Caution:** because this function allows you to output arbitrary HTML, you should [make sure it's trusted](https://en.wikipedia.org/wiki/Cross-site_scripting), and accessible.
```

This is a duplication from the `suggestion` template above. I don't mind repeating things 2 times and when we will need it a 3rd time we should create a separate heading in the README and link to it.